GVRf/Framework/src/org/gearvrf/GVRScreenshot3DCallback.java

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf;
+
+import java.util.List;
+
+import android.graphics.Bitmap;
+
+/**
+ * Callback interface for asynchronous 3D screenshot capture. The screenshot
+ * results will be passed to the caller as the parameter of onScreenCaptures.
+ */
+public interface GVRScreenshot3DCallback {
```

Comment:
Since I named the capture function "captureScreen3D" (which means "capture screenshot in 3D mode"), I also put "3D" later here. Also, in OpenGL, a 3D texture is called "GL_TEXTURE_3D". :)

If you do think putting 3D ahead is better, I can definitely change it accordingly.